packages/vanilla/src/complex/TableArrayControl.tsx

```diff
@@ -162,7 +162,7 @@ class TableArrayControl extends React.Component<
                               <DispatchField
                                 schema={schema}
                                 uischema={createControlElement(prop)}
-                                path={childPath}
+                                path={childPath + '.' + prop}
```

Comment:
shouldn't we do the same check here as in the material case? As far as I can see, this table is also used for primitive arrays

Reply:
We are inside a ternary expression here, see line 148